modules/nw-router-configuring-dual-stack.adoc

```diff
@@ -0,0 +1,92 @@
+// Module included in the following assemblies:
+//
+// * networking/routes/route-configuration.adoc
+
+[id="nw-router-configuring-dual-stack_{context}"]
+= Configuring the {product-title} Ingress Controller for dual-stack support
+
+When you install a {product-title} cluster on a IPv4/IPv6 dual-stack enabled bare metal infrastructure, your cluster to is externally reachable by {product-title} routes.
+
+The Ingress Controller automatically serves services that have both IPv4 and IPv6 endpoints, but you can configure the Ingress Controller for single-stack or dual-stack services.
```

Comment:
The first part of the sentence (automatically serves) makes it seem that the second part of the sentence isn't needed (configure for single or dual-stack networking).

A guess: Is it that services can be configured to expose IPv4 and IPv6 endpoints?  (That's a complete guess.)